Swift Code Challenge/Views/UserDetailsView.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  UserDetailsView.swift
+//  Swift Code Challenge
+//
+//  Created by Luiz Eduardo Mello dos Reis on 30/06/22.
+//
+
+import SwiftUI
+import MessageUI
+
+struct UserDetailsView: View {
+    @State var user: Person
+    var body: some View {
+        VStack{
+            MapView(lat: user.location.coordinates?.latitude, long: user.location.coordinates?.latitude)
```

Comment:
The maps don't display properly. They seem to not know what to display as the view seems to be in the middle of nowhere.